File: SumaMatrices.java
import java.util.Scanner;

public class SumaMatrices {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        final int n = 3;
        int[][] A = new int[n][n];
        int[][] B = new int[n][n];
        int[][] C = new int[n][n];

        System.out.println("Ingrese la primera matriz (3x3):");
        for (int i = 0; i < n; i++)
            for (int j = 0; j < n; j++)
                A[i][j] = sc.nextInt();

        System.out.println("Ingrese la segunda matriz (3x3):");
        for (int i = 0; i < n; i++)
            for (int j = 0; j < n; j++)
                B[i][j] = sc.nextInt();

        for (int i = 0; i < n; i++)
            for (int j = 0; j < n; j++)
                C[i][j] = A[i][j] + B[i][j];

        System.out.println("Resultado A + B:");
        for (int i = 0; i < n; i++) {
            for (int j = 0; j < n; j++)
                System.out.printf("%6d", C[i][j]);
            System.out.println();
        }
    }
}

File: SumaMatrices.c
#include <stdio.h>

#define N 3

int main() {
    int A[N][N], B[N][N], C[N][N];
    int i, j;

    printf("Ingrese la primera matriz (3x3):\n");
    for (i = 0; i < N; i++)
        for (j = 0; j < N; j++)
            scanf("%d", &A[i][j]);

    printf("Ingrese la segunda matriz (3x3):\n");
    for (i = 0; i < N; i++)
        for (j = 0; j < N; j++)
            scanf("%d", &B[i][j]);

    for (i = 0; i < N; i++)
        for (j = 0; j < N; j++)
            C[i][j] = A[i][j] + B[i][j];

    printf("Resultado A + B:\n");
    for (i = 0; i < N; i++) {
        for (j = 0; j < N; j++)
            printf("%6d", C[i][j]);
        printf("\n");
    }
    return 0;
}
